docs/index.rst

```diff
@@ -3,15 +3,91 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-ReX documentation
+ReX: Causal Responsibility Explanations for image classifiers
 =================
 
-Add your content using ``reStructuredText`` syntax. See the
-`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_
-documentation for details.
+**ReX** is a causal explainability tool for image classifiers.
+ReX is black-box, that is, agnostic to the internal structure of the classifier.
+We assume that we can modify the inputs and send them to the classifier, observing the output.
+ReX outperforms other tools on single explanations, non-contiguous explanations (for partially obscured images), and multiple explanations.
 
+.. image:: ../assets/rex-structure-600x129.png
+  :width: 600
+  :alt: ReX organisation
+
+For more information and links to the papers, see the :doc:`background` page.
+
+Installation
+----
+
+Clone the ReX repository and ``cd`` into it::
+
+   git clone git@github.com:ReX-XAI/ReX.git
+   cd ReX/
+
+We recommend creating a virtual environment to install ReX.
+ReX has been tested using versions of Python >= 3.10.
+The following instructions assume ``conda``::
+
+   conda create -n rex python=3.12
+   conda activate rex
+   pip install .
```

Comment:
Should this be switched to use poetry now?

Reply:
I don't think it _has_ to be - poetry installation instructions are in the CONTRIBUTING.md but a regular user wouldn't need to use poetry, they can install with whatever their usual workflow is. 

My personal feeling is that given the difficulties poetry's upper version pinning can cause, I'd rather not encourage regualr users to use it 😅 (even if we're using it as a build tool!)

Reply:
Agree with you that it doesn't have to be installed with poetry. But it feels unusual to be mixing and matching tools without a reason - if you want to keep it as is, could there be some docs somewhere explaining why? Maybe in the contributing file if that's where the poetry method is introduced?

Reply:
That's fair. I've added a brief explanation about why we use poetry in the contributing file!